src/trace-agent.js

```diff
@@ -66,7 +66,8 @@ function TraceAgent(config, logger) {
 TraceAgent.prototype.stop = function() {
   hooks.deactivate();
   pluginLoader.deactivate();
-  cls.destroyNamespace();
+  this.config.enabled = false;
+  this.policy = tracingPolicy.createTracePolicy(this.config);
```

Comment:
This is a roundabout way to get a policy that will never trace. Maybe we could add an explicit accessor for the `TraceNone` policy? Also, why is this policy change needed now that we've restored the unpatching of modules?